modules/running-distributed-data-science-workloads-from-notebooks.adoc

```diff
@@ -8,27 +8,77 @@ To run a distributed data science workload from a notebook, you must first provi
 .. In the "Clone a repo" dialog, enter `https://github.com/project-codeflare/codeflare-sdk.git` and then click *Clone*.
 The `codeflare-sdk` repository is listed in the left navigation pane.
 . Run a distributed workload job as shown in the following example:
 .. In the JupyterLab interface, in the left navigation pane, double-click *codeflare-sdk*.
 .. Double-click *demo-notebooks*, and then double-click *guided-demos*.
 .. Update each example demo notebook as follows:
-** Replace the links to the example community image with a link to your Ray cluster image.
+** Update the `import` section to import the `generate_cert` component:
++
+.Updated import section
+[source,bash]
+----
+from codeflare_sdk import generate_cert
+----
+
+** Replace the `default` namespace value with the name of your data science project.
+** Replace the link to the example community image with a link to your Ray cluster image.
+** Add the following Ray cluster authentication code after the Ray cluster creation step. Replace _your_cluster_name_ and _your_namespace_ with the values for your Ray cluster name and your data science project name.
++
+.Example Ray cluster authentication code
+[source,bash,subs="+quotes"]
+----
+generate_cert.generate_tls_cert(_your_cluster_name_, _your_namespace_)
+generate_cert.export_env(_your_cluster_name_, _your_namespace_)
+----
++
+[NOTE]
+====
+Mutual Transport Layer Security (mTLS) is enabled by default in the CodeFlare component in {productname-short}.
+Therefore, you must run the Ray cluster authentication code to create a secure connection between the Ray client that runs within a notebook and the Ray head client endpoint.
+====
+
+
+ifndef::upstream[]
+** If you have not configured a default local queue by including the `kueue.x-k8s.io/default-queue: 'true'` annotation as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-with-distributed-workloads_distributed-workloads#configuring-quota-management-for-distributed-workloads_distributed_workloads[Configuring quota management for distributed workloads], update the `ClusterConfiguration` code to specify the local queue for the Ray cluster, as shown in the following example:
++
+.Example local queue assignment
+[source,bash,subs="+quotes"]
+----
+local_queue="_your_local_queue_name_"
+----
+endif::[]
+ifdef::upstream[]
+** If you have not configured a default local queue by including the `kueue.x-k8s.io/default-queue: 'true'` annotation as described in link:{odhdocshome}/working-on-data-science-projects/#configuring-quota-management-for-distributed-workloads_distributed_workloads[Configuring quota management for distributed workloads], update the `ClusterConfiguration` code to specify the local queue for the Ray cluster, as shown in the following example:
++
+.Example local queue assignment
+[source,bash]
+----
+local_queue="your_local_queue_name"
+----
+endif::[]
+
 ** Set `instascale` to `False`.
```

Comment:
We have removed Instascale from the demo-notebooks so i think we should it''s mention from here as well

Reply:
Removed the Instascale references.